src/Controls/src/Core/Platform/Android/TabbedPageManager.cs

```diff
@@ -635,22 +635,27 @@ protected virtual ColorStateList GetItemTextColorStates()
 			}
 			else
 			{
+				// UnSelected tabs TextColor
 				if (barItemColor is not null)
```

Comment:
Here we repeat some code and can just create a new method:

```
int GetItemTextColor(Color? customColor, OriginalTabTextColors? originalColors)
{
    return customColor?.ToPlatform().ToArgb() ?? originalColors?.DefaultColor ?? 0;
}
```
To get a cleaner version: 
```
// UnSelected tabs TextColor
  defaultColor = GetItemTextColor(barItemColor, _originalTabTextColors);

  // Selected tabs TextColor
  checkedColor = GetItemTextColor(barSelectedItemColor, _originalTabTextColors);
```

Reply:
Thanks for the feedback. I have optimized the code as suggested @jsuarezruiz